src/components/NavBar/Nav.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from "react";
+import { AppBar, Toolbar, IconButton, Typography, Button, Dialog, TextField, DialogTitle, DialogContent, DialogContentText, DialogActions} from "@material-ui/core";
+import MenuIcon from '@material-ui/icons/Menu';
+import HomeIcon from '@material-ui/icons/Home';
+import "./styles.css"
+
+
+const NavBar = () => {
+  const [open, setOpen] = useState(false);
+  const [ money, setMoney] = useState(null);
+  const [ moneyVND, setMoneyVND] = useState(null)
+  const [ other, setOther] = useState('');
+  const request = {
+    "money": money,
+    "others": other,
+    "userId": '',
+  }
+
+  let VND = Intl.NumberFormat("en-VN", {
+    style: "currency",
+    currency: "VND",
+});
+  const handleClickOpen = () => {
+    console.log(localStorage.getItem("isLoggedIn"))
+    if (localStorage.getItem("isLoggedIn") === true) {
+    setOpen(true);
+    } else {
+      alert('Please log in to use this function.');
+      window.location.replace("/login")
+    }
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+  
+
+  const handleMoneyInput = (e) => {
+    setMoneyVND(VND.format(e.target.value));
+    setMoney(e.target.value)
+  }
+
+    // const handleRequest = (e) => {
+    //   e.preventDefault();
+    //   if (localStorage.getItem("isLoggedIn") === true) {
+    //      window.location.replace("/request")
+    //   } else {
+    //     alert('Please log in to use this function.')
+    //     window.location.replace("/login")
+    //   }
+    // }
+
+  
+
+    return (
+      <>
+        <AppBar position="static">
```

Comment:
this is going to fight with the navbar that @PatrickDrejerczak put together, I think

Reply:
nothing against material-ui, I think this could be better so we should just update the `HomePage` component to be sure to use this NavBar

Reply:
My NavBar is just the demo, not official since Pat is doing that. I just need sth to lead users to my login page, i wanna know their journey, like how they would reach the register page, is it the easy way to them... We will delete this nav bar when i have  Patrick's.